src/index.html

```diff
@@ -378,11 +378,90 @@ <h2 class="page__section-title">
         </div>
       </section>
 
-      <section class="page__section" id="contacts">
+      <section class="page__section contact-us" id="contacts">
         <div class="container">
           <h2 class="page__section-title">
             Contact us
           </h2>
+
+          <div class="grid grid--deskAndTablet">
+            <div class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5
+            ">
+              <form action="#">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="form-field contact-us__field"
+                >
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="form-field contact-us__field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                  form-field
+                  form-field--textarea
+                  contact-us__field
+                  contact-us__field--last
+                "></textarea>
+
+                <button class="button">
+                  Send
+                </button>
+              </form>
```

Comment:
under this button in the mobile version should be additional space between button and next block
<img width="349" alt="image" src="https://user-images.githubusercontent.com/125828769/223964566-d56985eb-0e2b-4f5b-880f-463d567ef0a0.png">